schematics/collection.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "../node_modules/@angular-devkit/schematics/collection-schema.json",
+  "schematics": {
+    "ng-add": {
+      "description": "Adds ng-bootstrap to the project and installs dependencies",
```

Comment:
I am hesitating here, should we remplace `ng-bootstrap` with `@ng-bootstrap/ng-bootstrap`?

Reply:
yeah, I was more thinking about the library name → `ng-bootsrtap`, maybe I should put "Adds ng-bootstrap **library** to the project"